concepts/Common/Integration Guides/40 Bind DevExtreme components to GraphQL/40 Bind DevExtreme components to GraphQL.md

```diff
@@ -0,0 +1,336 @@
+## What is GraphQL?
+
+GraphQL is a trending API query language and data processing layer. Facebook engineers created GraphQL because REST APIs did not meet their performance requirements.
+
+Modern websites need to display large amounts of data from more than one source. To get this data from a REST API, applications have to issue multiple network requests. Each request takes time to resolve, which slows the application down. If you don't optimize your SQL queries, API responses may include unnecessary data. The application wastes time loading data users don't get to see.
+
+GraphQL is an intermediary layer that takes a single API request, and responds to it with data from multiple sources. It allows you to get very specific, and does not include unnecessary data in its responses. You can replace your REST API with GraphQL, or use GraphQL alongside your REST API.
+
+For more information about the differences between GraphQL and REST, read the following article on the DevExtreme blog: [LINK].
+
+To try GraphQL for yourself, follow this tutorial.
+
+## Project overview
+
+This project consists of a client and a server.
+
+The client is a Vite/React Node.js application that displays a DevExtreme Data Grid. The server is an Express.js app that uses a GraphQL API to serve JSON data.
+
+The source code for this project is available [on GitHub](https://github.com/ivanblinov2k17/graphql). Refer to the repository if you get lost in the tutorial.
```

Comment:
I actually dislike the last sentence here. What do you mean "get lost in the tutorial"? Why do you set our customers to fail? It looks bad.